src/smolvm/cli/main.py

```
@@ -3205,6 +3286,212 @@ def _run_ssh(args: argparse.Namespace) -> int:
+    state_dir = Path.home() / ".smolvm" / "forwards"
+    state_dir.mkdir(parents=True, exist_ok=True)
+    return state_dir / f"{vm_id}.json"
+
+def _load_port_forwards(vm_id: str) -> list[dict]:
+    p = _port_forwards_path(vm_id)
+    if not p.exists():
+        return []
+    try:
+        import json
+        return json.loads(p.read_text())
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)